schedule/yast/autoyast_non_existing_graphical_target.yaml

```diff
@@ -0,0 +1,20 @@
+name:           autoyast_non_existing_graphical_target
+description:    >
+  Test AutoYaST installation when graphical target is specified as default in
+  AutoYaST control file, but the system does not have graphical environment
+  installed.
+  Covered cases:
+     - Verify that multi-user target is set as default in the case.
```

Comment:
Let's explain here that is fixed according to bug resolution, even if the package are not installed.

Reply:
Sorry, did not get what do you mean:)

Reply:
clarified with you offline, you still think is wrong behaviour and will keep in dev group until checking with systemd expert.